studio/components/interfaces/Home/ProjectUsageSection.tsx

```diff
@@ -43,7 +43,7 @@ const ProjectUsageSection: FC = observer(({}) => {
           <IconLoader className="animate-spin" size={14} />
           <p className="text-sm">Retrieving project usage statistics</p>
         </div>
-      ) : hasProjectData ? (
+      ) : true ? (
```

Comment:
@Ziinc this looks right?

Reply:
oops this is accidental, i thought  removed it 🤔